syntax = "proto3";
package lichtenstein.protocol.client;

/**
 * Provides information about node performance.
 */
message PerformanceInfo {

}